.golangci.yml

```diff
@@ -6,7 +6,7 @@ linters:
   - durationcheck
   - errcheck
   - exhaustive
-  - exportloopref
```

Comment:
This is being removed since it is deprecated?

Reply:
yes